swagger/actokids.yaml

```diff
@@ -15,10 +15,10 @@ tags:
 schemes:
   - https
 paths:
-  /event:
+  /events:
     post:
       tags:
-        - event
+        - Events
```

Comment:
nix the consume/produce xml here and throughout

Reply:
I think I got them all